sys-auth/fprintd/fprintd-0.8.1-r1.ebuild

```diff
@@ -18,11 +18,13 @@ IUSE="doc pam static-libs"
 RDEPEND="
```

Comment:
Increase revision when changing deps (just `git mv` it), so that people get it rebuilt. Otherwise, people with the old version installed will get libfprint upgraded and old slot removed.